test/test_helper.rb

```diff
@@ -79,11 +79,12 @@ def create_store_admin
 
   def login_store_admin
     create_store_admin
+    Store.first.update(approved: true)
 
     visit login_path
 
-    fill_in 'Username', with: 'storeadmin'
-    fill_in 'Password', with: 'password'
+    fill_in 'Username', with: 'Dave'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.